modules/ROOT/pages/6.3.2-release-notes.adoc

```diff
@@ -0,0 +1,42 @@
+= TinyMCE 6.3.2
+:navtitle: TinyMCE 6.3.2
+:description: Release notes for TinyMCE 6.3.2
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.3.2 was released for {enterpriseversion} and {cloudname} on Wednesday, March 1^st^, 2023. It includes {productname} 6.3.2. These release notes provide an overview of the changes for {productname} 6.3.2:
+
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:bug-fix[Bug fix]
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.3.2 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.3.2 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+[[bug-fix]]
+== Bug fix
+
+{productname} 6.3.2 also includes the following bug fix:
+
+=== Removed a workaround for ensuring stylesheets are loaded in an outdated version of webkit
+
+// https://ephocks.atlassian.net/browse/TINY-9433
```

Comment:
```suggestion
```

Out of sight out of mind

Reply:
I suggest to use only the ticket number, for example: TINY-9433, because this PR is visible to the public